pimd/pim_register.c

```diff
@@ -263,9 +266,70 @@ void pim_null_register_send(struct pim_upstream *up)
 			return;
 		}
 	}
-	pim_register_send((uint8_t *)&ip_hdr, sizeof(struct ip),
-			src, rpg, 1, up);
+	pim_register_send((uint8_t *)&ip_hdr, sizeof(struct ip), src, rpg, 1,
+			  up);
 }
+#else
+void pim_null_register_send(struct pim_upstream *up)
+{
+	struct ip6_hdr ip6_hdr;
+	struct pim_msg_header pim_msg_header;
+	struct pim_interface *pim_ifp;
+	struct pim_rpf *rpg;
+	pim_addr src;
+	unsigned char buffer[100];
+	struct ipv6_ph ph;
+
+	pim_ifp = up->rpf.source_nexthop.interface->info;
+	if (!pim_ifp) {
+		if (PIM_DEBUG_PIM_TRACE)
+			zlog_debug(
+				"%s: Cannot send null-register for %s no valid iif",
+				__func__, up->sg_str);
```

Comment:
Can you remove `__func__`? It's in progress to obsolete them.

Reply:
Addressed